### ISSUE TITLE
setting a new stop loss point

### ISSUE THREAD

**Issue by umutrcc** [freqtrade/freqtrade#5462]
I want the new stop point to be ema13 when the price goes above bbmid, how can I do this, I looked at the issue of advenced stop loss, but I couldn't solve it, I tried to do it myself in this way.

```


    stoploss = -0.02
    trailing_stop = False
    # ... populate_* methods

   class AwesomeStrategy(IStrategy):

    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        # <...>
        dataframe['ema13'] = ta.EMA(dataframe, timeperiod=13)   
        dataframe['bb_mid'] = bollinger2['mid']
    use_custom_stoploss = True

    def custom_stoploss(self, pair: str, trade: 'Trade', current_time: datetime,
                        current_rate: float, current_profit: float, **kwargs) -> float:

        dataframe, _ = self.dp.get_analyzed_dataframe(pair, self.timeframe)


        # Use parabolic sar as absolute stoploss price
       
        if current_profit > dataframe['bb_mid']:
            return stoploss_from_open(dataframe['ema13'], current_profit)


        # return maximum stoploss value, keeping current stoploss price unchanged
        return 1
        

```

but i got error.

**Comment by xmatthias** [comment]
`dp.get_analyzed_dataframe` returns a dataframe, not a single value.

You can't use `if` like this - but you'll need to access the last row of the dataframe for that.
The [advanced documentation](https://www.freqtrade.io/en/stable/strategy-advanced/#custom-stoploss-using-an-indicator-from-dataframe-example) contains a pretty good example for exactly this case.
